pkg/secrets/secrets.go

```diff
@@ -162,9 +162,15 @@ func SecretMounts(mountLabel, containerWorkingDir string, mountFile string) []rs
 	return secretMounts
 }
 
+func rchown(chowndir string, uid, gid int) error {
+	return filepath.Walk(chowndir, func(filePath string, f os.FileInfo, err error) error {
```

Comment:
Does this work? I know @umohnani8 had issues with filepath.Walk and symlinks, but I think we can guarantee the secrets dirs don't have symlinks...

Reply:
I think the userNS path should be used only to setup the bind mounts and by runc, everything else should be done in the same way as before